package haptix.comm.msgs;

/// \ingroup haptix_comm_msgs
/// \interface hxContactPoint_V
/// \brief Information about contacts.

import "hxContactPoint.proto";

message hxContactPoint_V
{
  repeated hxContactPoint contacts = 1;
}
